packages/calcite-components/src/components/select/select.scss

```diff
@@ -3,104 +3,112 @@
  *
  * These properties can be overridden using the component's tag as selector.
  *
+ * @prop --calcite-select-background-color: The background color of the component.
+ * @prop --calcite-select-border-color: The border color of the component.
  * @prop --calcite-select-font-size: The font size of `calcite-option`s in the component.
```

Comment:
I think we should keep the font-size and the space-inline internal. Also, adding a corner-radius that defaults to `sharp` and a box-shadow that defaults to `none` would help customization stories. Sorry, realize this is still in draft but I'll be OOO for a bit after today.

Reply:
`--calcite-select-font-size` and `--calcite-select-spacing` were added prior to the component token effort. I'll need to deprecate.